symphony-bdk-core/pom.xml

```diff
@@ -227,7 +233,8 @@
 
                         <!-- Exclude temporary classes from coverage -->
                         <exclude>com/symphony/bdk/core/SymphonyBdk.class</exclude>
-                        <exclude>com/symphony/bdk/core/service/**/*.class</exclude>
+                        <exclude>com/symphony/bdk/core/service/V4MessageService.class</exclude>
```

Comment:
why do we exclude this classes?

Reply:
These classes was created by @symphony-thibault and he left a comment on this class: 

> PLEASE PLEASE don't review this class !!